### ISSUE TITLE
Update the home page

### ISSUE THREAD

**Issue by lldelisle** [galaxyproject/galaxy-hub#2811]
Hi there,

I am not sure this is the good place to ask the question but...

This website is super important to engage people in the galaxy community. However, I noticed that the homepage is promoting an event that happened a month ago. In addition, I tested a lot of links among the public galaxy servers (https://galaxyproject.org/use/) and half of them do not exist anymore.

I think "Keeping this website up to date" should be an important task that should be attributed to one group of people working for galaxy...

**Comment by hrhotz** [comment]
Lucille and I are currently running a two days introductory workshop in Lausanne. 
As part of the general Galaxy project introduction yesterday, we felt bad pointing to the community hub 😞

**Comment by dannon** [comment]
@natalie-wa and @scottcain maybe?  I can help support technically, but I'd love for y'all to take the content part.

**Comment by scottcain** [comment]
Yeah, I was going to say the same. If the GTA image/href on the right can be removed and the site still look ok, I'd say let's just remove it for now.

The other minor thing I'll update shortly is the "Recent Publications" section, since the papers are all from 2023.

**Comment by natalie-wa** [comment]
It would also be great to get the socials added to the main page on the hub, at least the main Galaxy Project ones (LinkedIn, BlueSky, and Mastodon).

**Comment by scottcain** [comment]
@natalie-wa there are links to those in the footer (so, that is to say, almost completely hidden :-) I think the improvement here would be to put icons with links at the bottom of the "Galaxy is more than you think section." What do think of that?

**Comment by martenson** [comment]
we have https://github.com/martenson/public-galaxy-servers that can be used to verify whether servers declared as public actually are reachable

**Comment by lldelisle** [comment]
Indeed, thanks @martenson.

**Comment by bgruening** [comment]
@natalie-wa anything we can help here?
